swap_meet/item.py

```diff
@@ -1,2 +1,25 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=0):
+        if type(id) != int and id != None:
+            raise TypeError("id must be an int")
+        if not id:
+            id = uuid.uuid4().int
+        self.id = id
+        if type(condition) != int and type(condition) != float and condition != None:
+            raise TypeError("condition must be an int or a float")
+        self.condition = condition
+        if type(age) != int and type(age) != float and age != None:
+            raise TypeError("age must be an int or a float")
+        self.age = age 
```

Comment:
Very thorough type-checking! You probably don't want to allow `condition` to be `None`, since `condition_description` would fail if it were